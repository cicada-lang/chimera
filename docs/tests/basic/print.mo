print 1
print "a"
print ["a", "b", "c"]
print { a: 1, b: 2, c: 3 }
