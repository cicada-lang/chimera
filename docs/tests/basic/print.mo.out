1
"a"
["a", "b", "c"]
{ "a": 1, "b": 2, "c": 3 }
$Primitive(stringAppend, 2)
$Curried($Primitive(stringAppend, 2), 2, ["abc"])
$Primitive(equal, 2)
$Curried($Primitive(equal, 2), 2, ["abc"])
$Relation(Equal)
$Curried($Relation(Equal), 2, ["abc"])
() => {}
(x) => {
  return x
}
(x, y) => {
  return [x, y]
}
$Curried((x, y) => {
  return [x, y]
}, 2, [1])
